db/migrate/20220921141636_add_sharepoint_link_to_project.rb

```diff
@@ -0,0 +1,5 @@
+class AddSharepointLinkToProject < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :sharepoint_link, :string
```

Comment:
I know this is a total pain, but I have seen lots of evidence that there is an 'establishment' sharepoint folder and a 'trust' sharepoint folder, how do you feel about updating this to `establishment_sharepoint_link` now as we are here?